src/tags/channelpos.js

```diff
@@ -11,11 +11,20 @@ const Builder = require('../structures/TagBuilder');
 
 module.exports =
     Builder.APITag('channelpos')
-        .withDesc('Returns the position of the current channel.')
+        .withArgs(a => [a.optional('channelid')])
+        .withDesc('Returns the position of the current channel. If no channelid is given, the current channels position will be returned.')
         .withExample(
             'This channel is in position {channelpos}',
             'This channel is in position 1'
         )
         .whenArgs(0, async (_, context) => context.channel.position)
+        .whenArgs(1, async (_, context, args) => {
+            let i = 0;
+            context.guild.channels.foreach(g => {
```

Comment:
Would it be possible to use indexof here rather than a loop?

Reply:
Feel like it should be renamed to `{channels}` for consistency with its closest relative: `{roles}`

- [ ] Add `<channel> [quiet]` argument in `{channelid}`
    - Similar to `{roleid;<role>;[quiet]}`

- [ ] Rename `{channelpos}` to `{channels}`
    - Similar to `{roles}`


```cs
{indexof;
    {roles};
    {roleid;roleName}
}
```
> To find the specific position of a role.

```cs
{indexof;
    {channels};
    {channelid;channelName}
}
```
> To find the specific position of a channel.